files/__addonLocation__/.eslintrc.cjs

```diff
@@ -38,10 +35,10 @@ module.exports = {
 <% } %>    // node files
     {
       files: [
-        './.eslintrc.js',
+        './.eslintrc.cjs',
```

Comment:
what about `*.cjs`?

Reply:
> what about `*.cjs`? and `.prettierrc.js` should be cjs, too

only the files that I've touched here are delivered from this repo (eg https://github.com/embroider-build/addon-blueprint/blob/main/files/__addonLocation__/.eslintrc.cjs), the rest seem to be from the base addon
as far as I've generated the addon with this command it creates a `.prettierrc.js` and not a .cjs file hence I've left that as is
the changes here are correct as for the output of this blueprint (to the best of my ability to verify)

not sure what if anything you expect to be different here

Reply:
It's just that if we throw type=module in the package.json, the prettier file will be required to be cjs

Reply:
> It's just that if we throw type=module in the package.json, the prettier file will be required to be cjs

these are regex's so how about I make all options valid?

Reply:
How do you mean? All the 'node files' should be .cjs, and then this eslint entry can be `*.cjs` instead of the 4 or 5 lines it is now. But, i think bucause all our browser code is in src, we could probably safely assume all `*.{cjs,js}` are node. The toplevel cjs in def node, and the top leves js would be esm. One eslint entry to rule them all! (Of the package root lol)

Reply:
(tho, because parserType is 'script', only cjs is supported atm)